doc/cluster_spec.md

```diff
@@ -15,7 +15,9 @@ Some options in a running cluster specification can be changed to update the des
 |---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|-------------------|-------------------------------------------------------------------------------------------------------------------------------------|
 | sleepInterval             | interval to wait before next check (for every component: keeper, sentinel, proxy).                                                                                                                                                                                                                                                                                                                                                                                                | no                        | string (duration) | 5s                                                                                                                                  |
-| requestTimeout            | time after which any request (keepers checks from sentinel etc...) will fail.                                                                                                                                                                                                                                                                                                                                                                                                     | no                        | string (duration) | 10s                                                                                                                                 |
+| requestTimeout            | time after which any request (keepers checks from sentinel etc...) will fail. | no                        | string (duration) | 10s                     | 
+| proxyTimeoutInterval      | time after which a proxy will fail when communicating with database | no                        | string (duration) | 15s                               |
+| proxyCheckInterval        | proxy check interval | no                        | string (duration) | 5s                               |
```

Comment:
This is the interval where the proxy will check for updates from the store. It must be smaller then proxyTimeoutInterval.